content/docs/start/data-and-model-versioning.md

```diff
@@ -8,7 +8,7 @@ Git.'
 # Get Started: Data Versioning
 
 How cool would it be to make Git handle arbitrary large files and directories
-with the same performance as with small code files? Imagine doing a `git clone`
+with the same performance that you get with small code files? Imagine doing a `git clone`
```

Comment:
I agree that the current text could be improved but I'm not sure that the proposed change is much better.

Reply:
Also the paragraph wrapping (80 chars) is broken here 🙂 (no big deal since we can merge the Restyled PR #2508).